x/airdrop/module.go

```diff
@@ -115,23 +114,11 @@ func (am AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {
 	keeper.RegisterInvariants(ir, am.keeper)
 }
 
-// Route returns the message routing key for the airdrop module.
-func (am AppModule) Route() sdk.Route {
-	return sdk.Route{}
-}
-
 // QuerierRoute returns the airdrop module's querier route name.
```

Comment:
`QuerierRoute` is also removed in 0.47 afaik.

Reply:
Yes that's correct

Reply:
Sorry I think that could be wrong....